cromwell-instance-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       service: jmui
     command: ["npm", "run-script", "ng", "--", "serve", "--host", "jmui"]
     volumes:
+      - /ui/src/assets/environments/
```

Comment:
Create a volume for the below config file to be mounted into. I only 60% understand why this was needed - it's related to the fact that `ui/src` is already separately mounted in the compose file this one inherits from. Without this, the mounted config file was always empty.